### PR TITLE
Task 10 is ready and waiting for your corrections.

### PR DESCRIPTION
**Pull request by Denisyasyuchenya**
Mov-10 is awaiting your review. I'm starting to do mov-11.

### CODE REVIEW

File: src/main/resources/db/changelog/v-1.0/db.changelog-v.1.0.xml [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <preConditions>
+        <dbms type="mysql"/>
+    </preConditions>
+
+        <changeSet author="Денис" id="01">
```
Comment (RomanKashenok) [inline src/main/resources/db/changelog/v-1.0/db.changelog-v.1.0.xml]: Latin please

File: src/main/resources/db/changelog/v-1.0/db.changelog-v.1.0.xml [hunk 2]
```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <preConditions>
+        <dbms type="mysql"/>
+    </preConditions>
```
Comment (RomanKashenok) [inline src/main/resources/db/changelog/v-1.0/db.changelog-v.1.0.xml]: you don't need to check it if you alredy did it in master changelog

**Review by RomanKashenok — APPROVED** [review]
Fix minor comments and merge

File: src/main/resources/db/changelog/v-1.0/db.changelog-v.1.0.xml (hunk 2 above)
Comment (Denisyasyuchenya) [reply src/main/resources/db/changelog/v-1.0/db.changelog-v.1.0.xml]: ok

File: src/main/resources/db/changelog/v-1.0/db.changelog-v.1.0.xml (hunk 1 above)
Comment (Denisyasyuchenya) [reply src/main/resources/db/changelog/v-1.0/db.changelog-v.1.0.xml]: no problem)

**Pull request merged**